openslides/users/models.py

```diff
@@ -175,44 +175,7 @@ class Meta:
         ordering = ('last_name', 'first_name', 'username', )
 
     def __str__(self):
-        return self.get_full_name()
-
-    def get_full_name(self):
-        """
-        Returns a long form of the name.
-
-        E. g.: * Dr. Max Mustermann (Villingen)
-               * Professor Dr. Enders, Christoph (Leipzig)
-        """
-        structure = '(%s)' % self.structure_level if self.structure_level else ''
-        return ' '.join((self.title, self.get_short_name(), structure)).strip()
-
-    def get_short_name(self, sort_by_first_name=None):
-        """
-        Returns only the name of the user.
-
-        E. g.: * Max Mustermann
-               * Enders, Christoph
-        """
-        # Strip white spaces from the name parts
-        first_name = self.first_name.strip()
-        last_name = self.last_name.strip()
-
-        # The user has a last_name and a first_name
-        if first_name and last_name:
-            if sort_by_first_name is None:
-                sort_by_first_name = config['users_sort_users_by_first_name']
-            if sort_by_first_name:
-                name = ' '.join((first_name, last_name))
-            else:
-                name = ', '.join((last_name, first_name))
-
-        # The user has only a first_name or a last_name or no name
-        else:
-            name = first_name or last_name or self.username
-
-        # Return result
-        return name
+        return ' '.join((self.first_name, self.last_name))
```

Comment:
I changed it to old logic (although first AND last name can not be empty because you have to enter at least one name part. And the python __str__ method is now used for debugging and printing only, isn't it?)